docs/docs/layouts/modal.md

```diff
@@ -0,0 +1,17 @@
+A large size module with an overlay behind it - meant to cover a substantial area of the browser window, so that it demands attention from the user.
+
+## variant
+
+Variants determines any extra content that may be used by the module.
+
+| Value | Type | Behavior |
+|---|---|---|
+| 1 | int | `default` text-only module |
+| 2 | int | module includes an image |  
+| 3 | int | module includes a content recommendation |  
+
+### Image - [Live Preview](../../examples/preview/layouts/modal/image.html)
+
+![Image Slideout](../examples/img/layouts/modal/image.png)
+
+<pre data-src="../../examples/src/layouts/modal/image.js"></pre>
```

Comment:
`1` is the default so if they've looked at the types they've seen it before. I need to add `3`, I worked on these before the content recommendations got merged. :P